files/galaxy/tpv/destinations.yml.j2

```
@@ -210,7 +207,7 @@ destinations:
         - type: singularity
           shell: "/bin/bash"
           resolve_dependencies: true
-          identifier: "/data/0/singularity_base_images/centos:8.3.2011"
+          identifier: "/data/{{ dnb.0.path }}/singularity_base_images/centos:8.3.2011"
```

```suggestion
          identifier: "{{ dnb.0.path }}/singularity_base_images/centos:8.3.2011"
```

[According to the mounts repo](https://github.com/usegalaxy-eu/mounts/blob/2c31cf88d6db95a6f6ae924f1680631d250948a1/mountpoints.yml#LL60C18-L60C18), `{{ dnb.0.path }}` corresponds to `/data/0`.